lib/src/constants/constant_text.dart

```diff
@@ -1,25 +1,59 @@
 class AppText {
-  static const String buttonGreen = "scratch";
+  static const String iconText = 'scratch';
+  static const String fullNameErrorText = 'Fullname cannot be empty';
+
+  static const String emailErrorText =
+      'Please enter a valid email, e.g: john@gmail.com';
+  static const String passwordErrorText =
+      'Password must have as least 8 characters, including numbers, uppercase, lowercase characters and uncharacters';
+  static const String userAlreadyExist = 'User is already exist!';
+
+  static const String emailErrorLoginText = "Email incorrect";
+  static const String passwordErrorLoginText = "Password incorrect";
+
+  static RegExp emailRegex = RegExp(
+      r'^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$');
+  static RegExp passwordRegex =
+      RegExp(r'^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$');
```

Comment:
should we move these regs to another class?

Reply:
Done